settings.gradle

```diff
@@ -1,2 +1,3 @@
+include ':shoplist'
```

Comment:
consider moving the module declaration after app declaration. nothing in particular just preference